new-docs/en/ecosystem/external-table/hive-of-doris.md

```diff
@@ -104,16 +105,20 @@ The supported Hive column types correspond to Doris in the following table.
 |   DECIMAL  | DECIMAL |  |
 |   DATE   |  DATE  |                                   |
 |   TIMESTAMP  | DATETIME | Timestamp to Datetime will lose precision |
+|  BINARY   |  BITMAP  |  |
```

Comment:
need another white-space before BINARY? It seams there are 4 white-space before 'TIMESTAMP', but 3 white-space before 'BINARY'.

Reply:
Thank you. I've added a white-space